pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -212,66 +237,84 @@ def compile_dir(dfn):
     '''
     Compile *.py in directory `dfn` to *.pyo
     '''
+
+    if get_bootstrap_name() != "sdl2":
+        # HISTORICALLY DISABLED for other than sdl2. NEEDS REVIEW! -JonasT
```

Comment:
what's up with this?

Reply:
I retained the exact behavior pre-merge, which is that it's disabled for webview and service_only. As the comment says it might be a good idea to look at this and maybe enable it for all, or figure out why - I simply considered this out of the scope of the merge which is mostly supposed to just keep the current behavior for now, but get rid of the duplicate files

Reply:
OK it makes sense with that explanation. Perhaps you can update the comment to just what you said here

Reply:
I thought with the `historically disabled` it'd be obvious it needs to get looked at & refactored some time soon :grimacing: but I can add some more explanation if you think that'd be helpful